src/components/Content/Content.js

```diff
@@ -1,15 +1,214 @@
-import React from "react";
+import React, { useState } from "react";
+import DataService from "../../services/DataService";
+import { FormGroup, Label, Input, InputGroup, InputGroupAddon, InputGroupText } from "reactstrap";
 
 import "./Content.css";
 
+const formatSections = (data, activeSection, activeSubsection, setActiveSubsection, variables, setVariables) => {
+  if (data == null) {
+    return;
+  }
+  return Object.entries(data).map(section => {
+    const key = section[0];
+    const value = section[1];
+
+    // If Parameters, export both A and B
+    if (key === "parameters") {
+      const parametersA = [key+"-a", value];
+      const parametersB = [key+'-b', value];
+
+      return (
+        <React.Fragment key="parameters">
+          {formatSection(parametersA, activeSection, activeSubsection, setActiveSubsection, variables, setVariables)}
+          {formatSection(parametersB, activeSection, activeSubsection, setActiveSubsection, variables, setVariables)}
+        </React.Fragment>
+      );
+    } else {
+      return formatSection(section, activeSection, activeSubsection, setActiveSubsection, variables, setVariables);
+    }
+  });
+};
+
+const formatSection = (section, activeSection, activeSubsection, setActiveSubsection, variables, setVariables) => {
+  const key = section[0];
+  const value = section[1];
+
+  let labelKey = "label";
+  let contentKey = "content";
+  if (key === "parameters-b") {
+    labelKey = "labelB";
+    contentKey = "contentB";
+  }
+  return (
+    <FormGroup key={key} className={`fieldset ${activeSection === key ? "active" : ""}`}>
+      <h2>{value[labelKey]}</h2>
+      <p>{value[contentKey]}</p>
+
+      <div className="subNavigation">
+        {Object.entries(value.fields).map(field => {
+          const key = field[0];
+
+          return (
+            <button 
+              key={key}
+              onClick={() => setActiveSubsection(key)}
+              className={key === activeSubsection ? "active" : "" } 
+            >
+              {key}
+            </button>
+          );
+        })}
+      </div>
+      <div className="subNavigationContent">
+        {Object.entries(value.fields).map(field => {
+          const key = field[0];
+          const value = field[1];
+
+          if (activeSubsection !== key) {
+            return null;
+          }
+
+          return (
+            <section key={key} className="Questions">
+              {Object.entries(value).map(question => {
+                return formatQuestion(question, variables, setVariables);
+              })}
+            </section>
+            );
+          })
+        }
+      </div>
+    </FormGroup>
+  );
+}
+
+const formatNavigation = (data, activeSection, setActiveSection) => {
+  if (data == null) {
+    return;
+  }
+  return Object.entries(data).map(section => {
+    let key = section[0];
```

Comment:
Use keys instead of entries
```suggestion
  return Object.keys(data).map(key => {
```